webpack/assets/javascripts/react_app/redux/actions/common/forms.js

```diff
@@ -49,36 +33,21 @@ const verifyProps = (item, values) => {
 };
 
 export const submitForm = ({
-  item, url, values, csrfToken = getcsrfToken(), method = 'post',
+  item, url, values,
 }) => {
   verifyProps(item, values);
   return dispatch =>
-    fetch(url, {
-      credentials: 'include',
-      method,
-      headers: {
-        'Content-Type': 'application/json',
-        Accept: 'application/json',
-        'X-CSRF-Token': csrfToken,
-      },
-      body: JSON.stringify(values),
-    })
-      .then(checkErrors)
-      .then(response =>
-        response
-          .json()
-          .then(body =>
-            dispatch({
-              type: `${item.toUpperCase()}_FORM_SUBMITTED`,
-              payload: { item, body },
-            }))
-          .then(() =>
-            dispatch(addToast({
-              type: 'success',
-              // eslint-disable-next-line no-undef
-              message: Jed.sprintf('%s was successfully created.', __(item)),
-            }))))
-      .catch((error) => {
-        throw error;
-      });
+    API.post(url, values)
```

Comment:
the reason i kept the method in the function header, is that we would like to use the same code path for updating an existing form too.